### PR TITLE
Convert oqs-kem-info.md code points to hex

### PR DESCRIPTION
**Pull request by WillChilds-Klein**
> Please give a brief explanation of the purpose of this pull request.

Minor documentation update to use hexadecimal notation for KEM code points.

> Does this PR resolve any issue?  If so, please reference it using automatic-closing keywords like "Fixes #123."

Not that I'm aware of.

---

# Notes

The initial motivation for this PR was a [small documentation change](https://github.com/open-quantum-safe/oqs-provider/pull/188/commits/261bc2fb06b02c9732f22c3f695a42e700855332). That change uncovered an unexpected warning message. It appears that the erroneous warning message motivating this commit is due to some vestigial code special-casing x-curves. The `nid_ecx_hyybrid` kem field was only used to generate the duplicate ecx nid warning message, so we track duplicates in a local `set` instead. Additionally, we expand the warning message to all curve NIDs, not just x-curves and treat duplicates as fatal because extra curve NIDs should be unique for each KEM.

Duplicate NID detection is tested below:

```
$ git diff
diff --git a/oqs-template/generate.yml b/oqs-template/generate.yml
index 4865eb7..fa3ddfe 100644
--- a/oqs-template/generate.yml
+++ b/oqs-template/generate.yml
@@ -114,6 +114,8 @@ kems:
           nid: '0x2F90'
         - hybrid_group: "x25519"
           nid: '0x6399'
+        - hybrid_group: "p521"
+          nid: '0x6399'
         - hybrid_group: "p256"
           nid: '0x639A'
       old:

$ LIBOQS_SRC_DIR=$(pwd)/../liboqs python3 oqs-template/generate.py
ERROR: duplicate hybrid NID for kyber768 : 0x6399 in generate.yml. Curve NIDs may only be specified once per KEM.

$ echo $?
1
```

I'm not sure what's up with the CI; I don't appear to have permissions to view the build errors in CircleCI. I ran the tests locally on a linux box:

```
$ cmake --build build && ( cd build; ctest -V )
...
100% tests passed, 0 tests failed out of 5

Total Test time (real) =  29.85 sec
```

### CODE REVIEW

**Review by baentsch — CHANGES_REQUESTED** [review]
This is a change to an automatically generated file and as such won't persist. Please change at the master file (`oqs-template/generate.yml`), re-generate (`python3 oqs-template/generate.py`) and re-run all tests.

**Comment by baentsch** [comment]
Thanks for the update: This is now perfect to retain.

> some guidance on that warning's original intent

This is basically a question/reminder to @bhess to re-evaluate the original ECX logic (via an extra-YML file) he originally introduced. IMO it could be removed as #177 completely changed the build logic to permit more than a single X-hybrid for the same PQ alg.

**Review by baentsch — APPROVED** [review]

File: oqs-template/generate.py [hunk 1]
```diff
@@ -193,25 +193,24 @@ def load_config(include_disabled_sigs=False):
...
+               extra_hybrid['bit_security'] = 192
+            if extra_hybrid['hybrid_group'] == "p521":
+               extra_hybrid['bit_security'] = 256
+            kem['hybrids'].append(extra_hybrid)
+            if 'hybrid_group' in extra_hybrid:
+                extra_hybrid_nid = extra_hybrid['nid']
+                if extra_hybrid_nid in hybrid_nids:
+                    print("Warning, duplicate hybrid NID for",
+                          kem['name_group'], ":", extra_hybrid_nid, "in generate.yml.")
+                    exit(1)
```
Comment (baentsch) [inline oqs-template/generate.py:212]: This `exit` warrants a bit more than a "Warning" message, no? I'd suggest to either remove the exit() or rephrase the Warning to "Fatal Error" (best with explanation what to change in the YML to get the generator to run again).

Comment (WillChilds-Klein) [reply oqs-template/generate.py:212]: Fair enough. This seems like it should be a fatal condition so I'll update the message to reflect. Please correct me if this is not the case.

**Comment by baentsch** [comment]
@WillChilds-Klein Thanks again for the contribution! CI has run successfully so we'll merge this tomorrow. As this is more than a trivial change, please consider amending this PR before merge by adding yourself to the Contributors list in the README.

**Review by dstebila — APPROVED** [review]

**Pull request merged**